test/Configure/is_conftest/fixture/SConstruct

```diff
@@ -0,0 +1,24 @@
+"""
+Test the nodes are created as conftest nodes in configure tests.
+"""
+import sys
+DefaultEnvironment(tools=[])
+
+env = Environment()
+
+conf = Configure(env)
+if sys.platform == "win32":
+    conf.env.Append(
+        CCFLAGS="/DEBUG /Z7 /INCREMENTAL:NO",
+        LINKFLAGS="/DEBUG /INCREMENTAL:NO",
+        PDB='${TARGET.base}.pdb')
+if not conf.TryRun("int main( int argc, char* argv[] ){return 0;}", '.c'):
+    print("FAIL")
+
+env = conf.Finish()
+
+for node in env.Glob('.sconf_temp/*'):
```

Comment:
probably should set the conf_dir explicitly to make this test a bit more future proof?

Reply:
won't this fail for the .ilk files we discussed previously?

Reply:
You can get the config dir from conf via `conf.confdir` for use in your Glob..

Reply:
> won't this fail for the .ilk files we discussed previously?

incremental linking is set to NO, so msvc wont generate ilk files.

Reply:
updated to use conf.confdir

Reply:
> updated to use conf.confdir

Did you push that?

Reply:
It's also an environment var, no?  https://scons.org/doc/production/HTML/scons-man.html#cv-CONFIGUREDIR

Reply:
> > updated to use conf.confdir
> 
> Did you push that?

yeah f21e361